internal/app/machined/pkg/controllers/network/route_status.go

```diff
@@ -122,6 +122,10 @@ func (ctrl *RouteStatusController) Run(ctx context.Context, r controller.Runtime
 				status.Protocol = nethelpers.RouteProtocol(route.Protocol)
 				status.Flags = nethelpers.RouteFlags(route.Flags)
 
+				if route.Attributes.Metrics != nil {
+					status.MTU = route.Attributes.Metrics.MTU
+				}
```

Comment:
also need an `else` clause:

```go
} else { 
   status.MTU = 0
}
```
if the MTU is removed (and attributes are nil), we should reset the value to 0